### PR TITLE
Updates for AMP/realms

### PR DESCRIPTION
**Pull request by emaciupe**
[ch34250]

### CODE REVIEW

**Review by domdinicola — APPROVED** [review]

File: src/etools_datamart/apps/mart/data/models/audit_financial_finding.py [hunk 1]
```diff
@@ -24,9 +24,6 @@ def process_country(self):
             except Audit.DoesNotExist:
                 pass
 
-    def get_title(self, record, values, field_name):
```
Comment (domdinicola) [inline src/etools_datamart/apps/mart/data/models/audit_financial_finding.py:27]: no needed?

Comment (emaciupe) [reply src/etools_datamart/apps/mart/data/models/audit_financial_finding.py:27]: it is not used as it works without - that's why there is no mapping for it

**Comment by emaciupe** [comment]
old models have been removed from etools - requires schema update

**Comment by domdinicola** [comment]
eTools AMP has been released? @emaciupe

**Pull request merged**